### ISSUE TITLE
The accuracy of GLMIM pre-trained model on LRW-1000

### ISSUE THREAD

**Issue by Xiaokeai18** [xing96/MIM-lipreading#1]
Hello,
Thanks for your fantastic work.
I'm trying to reproduce the paper so that I cloned your repo and run 'test' on the pre-trained model you offerd in README.
The model name is 'Baseline_GLMIM_acc_0.3879.pt'.
The mean accuracy on LRW-1000 is around 24%, lower than the paper. I dont know whether I made any mistake during testing.
So can you tell me the accuracy of the pre-trained model on LRW-1000?  Watching for your reply.

**Comment by Xiaokeai18** [comment]
Is the number in the filename (i.e. 0,3789) represents the real accuracy?

**Comment by Xiaokeai18** [comment]
Ohh, I found the reason. The new version of scypy.ndimage is deprecated, so I used the cv2.imread, and the cv2 uses BGR as default channel. I swiched to imageio.imread, and the problem is fixed.
Thanks a lot !

**Comment by aflyingnoob** [comment]
Thank you very much，while I still find a error at dataset.py, and I got the 38.79 after testing the Baseline_GLMIM.
